sdk/core/System.ClientModel/Directory.Build.props

```diff
@@ -0,0 +1,8 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Are we confident this is the correct boilerplate for this library?

Reply:
Copied from System.Memory.Data let me know if its incorrect.

Reply:
Why is it different from the one used by Azure.Core?

Reply:
I followed a System.* library instead of Azure.Core because this isn't an Azure library.  It mirrors what you did for the first beta.

Do we think this should block beta?

Reply:
My concern was just that I don't have a good understanding of what's in these files, and because of that, I don't know how reversible shipping a beta with a client-library one of these instead of a Core one of these would be.  But it looks like you've adopted the Core version now?  If so, I'm happy to resolve this comment and thank you.

Reply:
Ah, looks like it still differs from the Azure.Core one: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/Directory.Build.props

Is there reason not to use the Azure.Core one instead?  This one sets `<IsClientLibrary>` to true, which I'm not sure we want.